awx/ui_next/src/components/LaunchPrompt/steps/useCredentialsStep.jsx

```diff
@@ -19,7 +19,7 @@ export default function useCredentialsStep(
     initialValues: getInitialValues(config, resource),
     validate,
     isReady: true,
-    error: null,
+    contentError: null,
```

Comment:
why did this one not get the same pattern as the other steps (there's no formError key)

Reply:
I will add the `formError` key returning `null`. There is another issue related to the validation of credentials https://github.com/ansible/awx/issues/7018